storybook/storybook-docs/src/color.stories.mdx

```diff
@@ -0,0 +1,35 @@
+import { Meta } from "@storybook/blocks";
+import { ColourPalette } from "./components/ColourPalette";
```

Comment:
Did you choose not to use [Storybooks ColorPalette](https://storybook.js.org/docs/api/doc-block-colorpalette) on purpose?

Reply:
Yes, my first attempt used it. Its design doesn’t match ours, we don’t have swatches of shades and tints of one hue, and no names to display on the left.

I’ll be replacing their IconGallery with one of our own as well.